README.md

```diff
@@ -131,11 +131,15 @@ quality_report = evaluate_quality(
 ```
 
 ```
-Creating report: 100%|██████████| 4/4 [00:00<00:00, 19.30it/s]
-Overall Quality Score: 89.12%
-Properties:
-Column Shapes: 90.27%
-Column Pair Trends: 87.97%
+Generating report ...
+
+(1/2) Evaluating Column Shapes: |████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9/9 [00:00<00:00, 1133.09it/s]|
```

Comment:
The SDV output is this long of a progress bar, but I can always cut this down to something more reasonable!

Reply:
Yeah I'd recommend just deleting a few of the solid black squares to shorten the length of the bar